### PR TITLE
feat(RESTJSONErrorCodes): add error code 50086

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

**Reference Discord API Docs PRs or commits:**
- https://github.com/discord/discord-api-docs/pull/4302

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Comment by suneettipirneni** [comment]
upstream PR merged

**Comment by vladfrangu** [comment]
Flag was renamed, please update accordingly

File: deno/rest/common.ts [hunk 1]
```diff
@@ -170,6 +170,7 @@ export enum RESTJSONErrorCodes {
 	InvalidActionOnArchivedThread = 50083,
 	InvalidThreadNotificationSettings,
 	ParameterEarlierThanCreation,
+	CommunityServerChannelMustBeTextChannel,
```
Comment (almeidx) [inline deno/rest/common.ts:173]:
```suggestion
	CommunityServerChannelsMustBeTextChannels,
```

File: rest/common.ts [hunk 2]
```diff
@@ -170,6 +170,7 @@ export const enum RESTJSONErrorCodes {
 	InvalidActionOnArchivedThread = 50083,
 	InvalidThreadNotificationSettings,
 	ParameterEarlierThanCreation,
+	CommunityServerChannelMustBeTextChannel,
```
Comment (almeidx) [inline rest/common.ts:173]:
```suggestion
	CommunityServerChannelsMustBeTextChannels,
```

File: deno/rest/common.ts (hunk 1 above)
Comment (vladfrangu) [inline deno/rest/common.ts:173]: Channels*

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Review by almeidx — CHANGES_REQUESTED** [review]
To mirror the docs text

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**